src/components/reusuables/cards/ServiceCard.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { StaticImage } from "gatsby-plugin-image"
+import { Link } from "gatsby"
+
+const ServiceCard = (props) => {
+  return (
+    <div className="service-card">
+      <Link className="service-card-link">
+        <div className="hover-bg"></div>
+        {props.number ? (
```

Comment:
Can we set this logic at the top like we discussed in our call? Change true to boolean instead of string.

cont number = number === true ? number : false;
{number && (



)
}

Reply:
Logic has been updated